tree/dataframe/src/RDataFrame.cxx

```diff
@@ -945,14 +981,14 @@ auto f = d.Filter("myFriend.MyCol == 42");
 
 Friend TTrees with a TTreeIndex are supported from ROOT v6.24.
 
-### Reading file formats different from ROOT's
-RDataFrame can be interfaced with RDataSources. The RDataSource interface defines an API that RDataFrame can use to read arbitrary data formats.
+### <a name="other-file-formats"></a>Reading file formats other than ROOT's
```

Comment:
This is true but slightly misleading because there is an RDataSource defined for RNTuple, which is still a ROOT data format. Maybe better this way ?
```suggestion
### <a name="other-file-formats"></a>Reading file formats other than ROOT TTree
```

Reply:
I agree that's better but it should be "data formats" then -- will fix